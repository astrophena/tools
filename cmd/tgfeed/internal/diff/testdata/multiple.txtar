-- a --
hello world

foo bar

SAMPLE TEXT
sample text
Sample text
-- b --
hello
world

foo bar

SaMPLE TEXT
saMple text
SamPle text
