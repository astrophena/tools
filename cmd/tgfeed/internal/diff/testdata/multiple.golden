diff a b
--- a
+++ b
@@ -1,7 +1,8 @@
-hello world
+hello
+world
 
 foo bar
 
-SAMPLE TEXT
-sample text
-Sample text
+SaMPLE TEXT
+saMple text
+SamPle text
